siliconcompiler/client.py

```diff
@@ -51,16 +51,16 @@ def remote_preprocess(chip):
     active = manager.dict()
 
     # Setup up tools for all local functions
-    remote_steplist = chip.getkeys('flowgraph')
+    remote_steplist = chip.getkeys('flowgraph',chip.get('flow'))
```

Comment:
Normally I wouldn't comment on missing spaces, but the 'suggest changes' feature makes it easy to address these nitpicks quickly.

```suggestion
    remote_steplist = chip.getkeys('flowgraph', chip.get('flow'))
```